containers/cart-page/index.css

```diff
@@ -0,0 +1,26 @@
+.quantity-section > * {
+  background-color: transparent;
+}
+
+.increase,
+.decrease {
```

Comment:
use more descriptive names like `increase-items` and `decrease-items`

Reply:
yes, will change this